perfkitbenchmarker/benchmark_sets.py

```diff
@@ -190,7 +191,7 @@ def GetBenchmarksFromFlags():
     else:
       benchmark_names.add(benchmark)
 
-  # Expand recursive sets
+    # Expand recursive sets
```

Comment:
Please don't change the comment indentations like this. If you're having problems with flake8, please install the following versions:
```
pip install pep8==1.5.7
pip install flake8==2.4.1
```
#979 should prevent future problems when it makes its way into the master branch.